config/fedora.json

```diff
@@ -6,9 +6,20 @@
       "com_redhat_component",
       "summary",
       "version",
-      "usage"
+      "usage",
+      "release"
     ],
-    "optional": []
+    "optional": [
+      "architecture",
+      "url",
+      "help",
+      "build-date",
+      "distribution-scope",
+      "vcs-ref",
```

Comment:
vcs-ref is not mentioned at all. The same below.

Reply:
It's [general label](https://github.com/projectatomic/ContainerApplicationGenericLabels) -- we should probably add them to our document.